controllers/clusteroperator_controller.go

```diff
@@ -180,6 +191,33 @@ func getResources(infra *configv1.Infrastructure) []client.Object {
 	return nil
 }
 
+func (r *CloudOperatorReconciler) composeConfig(ctx context.Context) (operatorConfig, error) {
+	config := operatorConfig{}
+
+	infra := &configv1.Infrastructure{}
+	if err := r.Get(ctx, client.ObjectKey{Name: infrastructureName}, infra); err != nil {
+		klog.Errorf("Infrastructure cluster could not be fetched", err)
+		return config, err
+	}
+
+	images, err := getImagesFromJSONFile(r.ImagesFile)
+	if err != nil {
+		klog.Errorf("Unable to decode images file from location %s", r.ImagesFile, err)
+		return config, err
+	}
+
+	_, err = getProviderFromInfrastructure(infra)
+	if err != nil {
+		klog.Errorf("Unable to decode images file from location %s", r.ImagesFile, err)
+		return config, err
+	}
```

Comment:
What's this? Do we not need the platform value? Also, if we already have this in the parent scope, why not pass it in?

Reply:
Some leftovers from testing :sob: Very inconvenient when manifests from the platform I'm using are not merged: https://github.com/openshift/cluster-cloud-controller-manager-operator/pull/15